src/Administration/Resources/app/administration/src/module/sw-settings-language/page/sw-settings-language-list/index.js

```diff
@@ -73,38 +73,22 @@ export default {
                     label: 'sw-settings-language.list.columnName',
                     dataIndex: 'name',
                     inlineEdit: true,
-                    allowResize: true,
-                },
-                {
-                    property: 'id',
-                    label: 'sw-settings-language.list.columnDefault',
-                    align: 'center',
-                    allowResize: true,
-                },
-                {
-                    property: 'active',
-                    dataIndex: 'active',
-                    label: 'sw-settings-language.list.columnActive',
-                    inlineEdit: 'boolean',
-                    allowResize: true,
-                    align: 'center',
                 },
                 {
                     property: 'locale',
                     dataIndex: 'locale.id',
                     label: 'sw-settings-language.list.columnLocaleName',
-                    allowResize: true,
                 },
                 {
                     property: 'translationCode.code',
                     label: 'sw-settings-language.list.columnIsoCode',
-                    allowResize: true,
                 },
                 {
-                    property: 'parent',
-                    dataIndex: 'parent.id',
-                    label: 'sw-settings-language.list.columnInherit',
-                    allowResize: true,
+                    property: 'active',
+                    dataIndex: 'active',
+                    label: 'sw-settings-language.list.columnActive',
+                    inlineEdit: 'boolean',
+                    align: 'center',
```

Comment:
Being curious, would you be so kind to provide a screenshot for the current state? 🙂